code/modules/reagents/chemistry/machinery/chem_master.dm

```diff
@@ -20,9 +20,20 @@
 	var/screen = "home"
 	var/analyzeVars[0]
 	var/useramount = 30 // Last used amount
+	var/list/pillStyles = null
 
 /obj/machinery/chem_master/Initialize()
 	create_reagents(100)
+
+	//Calculate the span tags and ids fo all the available pill icons
+	var/datum/asset/spritesheet/simple/assets = get_asset_datum(/datum/asset/spritesheet/simple/pills)
```

Comment:
is it inefficient to fetch this a lot? otherwise I can just cache it on the chem master

Reply:
`get_asset_datum` is a trivial lookup and could easily be made even more trivial w/ slight tweaks.